src/modules/core/utils/time.utils.ts

```diff
@@ -214,22 +215,28 @@ export const formatMinutesDuration = (minutes: number): string => {
 export const emptyTimeRange = formatTimeRange(new Time(), new Time());
 
 // formatter duration
-export const formatDuration = (duration: number): string => {
+export const formatDuration = (duration: number, intl?: IntlShape): string => {
   const hours = Math.floor(duration / 60);
   const minutes = duration % 60;
 
   let formattedDuration = '';
 
   if (hours > 0) {
-    formattedDuration += `${hours} ${hours === 1 ? 'hour' : 'hours'}`;
+    formattedDuration += `${hours} ${intl?.formatMessage(
+      { id: hours === 1 ? 'duration.hour' : 'duration.hours' },
+      { count: hours }
```

Comment:
Since you are using intl you can use pluralization insted of selecting id here is the [docs](https://next-intl-docs.vercel.app/docs/usage/messages#cardinal-pluralization).